code/game/machinery/suit_storage_unit.dm

```diff
@@ -385,6 +390,9 @@
 		if(uv_super)
 			visible_message(span_warning("[src]'s door creaks open with a loud whining noise. A cloud of foul black smoke escapes from its chamber."))
 			playsound(src, 'sound/machines/airlock_alien_prying.ogg', 50, TRUE)
+			var/datum/effect_system/smoke_spread/bad/black/smoke = new
```

Comment:
you uh, you sure you want the effects of this?

Reply:
/bad/ just means that it causes coughing, which seems fitting to opening something that just absolutely melted stuff that you PROBABLY shouldnt be inhaling. Plus its only on the same tile as the machine itself. So I dont see a reason not to;

plus this adds a bad/black subtype for future use

Reply:
Doesn't it make you drop your items? That's most of what I was worried about. being on one tile is less concerning

Reply:
Yeaaaa it does, just saw it. But again, being on one tile and doing that more just means if anyone goes to take a DEEP whiff of that molten plasteel then they'll cough-drop it, which again makes sense.
If you want, I can move it back to a normal black-colored smoke though